Sprint-3/3-mandatory-practice/implement/count.js

```diff
@@ -1,5 +1,10 @@
 function countChar(stringOfCharacters, findCharacter) {
-    return 5
+  let count = 0;
+  for (let i = 0; i < stringOfCharacters.length; i++) {
+    if (stringOfCharacters[i] === findCharacter) count++;
+  }
```

Comment:
This passes the tests, which means that it's correct code according to our definition!